### PR TITLE
Add Manifesto to wiki

### LINKED ISSUE #329: Move Aragon Manifesto into the wiki
Per AGP-0 discussion. This way people can propose changes to it as the network evolves.

### PR DESCRIPTION
**Pull request by john-light**
Closes #329

### CODE REVIEW

File: docs/documentation/manifesto.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+# The Aragon Manifesto
+
+Version 1.0
+
+_Version 1.0 of the Aragon Manifesto was originally published on the Aragon blog [here](https://blog.aragon.org/the-aragon-manifesto-4a21212eac03/)._
```
Comment (Smokyish) [inline docs/documentation/manifesto.md]: Should be Aragon Project blog

File: docs/documentation/manifesto.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+## A pledge to fight for freedom
+
+**We believe the fate of humanity will be decided at the frontier of technological innovation.**
+
+We will either see technology lead to a more free, open, and fair society or reinforce a global regime of centralized control, surveillance, and oppression. Our fear is that without a global, conscious, and concerted effort, the outlook is incredibly bleak.
+
+The Internet has opened the doors for universal, cross-border, and non-violent collaborative effort to [fight for our freedom](https://www.youtube.com/watch?v=AqjIWmiAidw).
```
Comment (Smokyish) [inline docs/documentation/manifesto.md]: `fight for freedom` is missing bolding

File: docs/documentation/manifesto.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
+# The Aragon Manifesto
+
+Version 1.0
+
+_Version 1.0 of the Aragon Manifesto was originally published on the Aragon blog [here](https://blog.aragon.org/the-aragon-manifesto-4a21212eac03/)._
+
+## A pledge to fight for freedom
```
Comment (Smokyish) [inline docs/documentation/manifesto.md]: In the blog post this is a subtitle not a header

File: docs/documentation/manifesto.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+**We believe the fate of humanity will be decided at the frontier of technological innovation.**
```
Comment (Smokyish) [inline docs/documentation/manifesto.md]: In the blog post this is a header, not bolded text

File: docs/documentation/manifesto.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+Building tools to create and manage decentralized organizations will unleash a [Cambrian explosion](https://en.wikipedia.org/wiki/Cambrian_explosion) of new governance forms, and the competition among them will raise the bar globally.
+
+It will finally allow us to experiment with governance at the speed of software.
+
+Instead of complaining about how badly incentives are set in the world and how poorly resources are allocated, we will have the power to create systems that better align incentives and distribute resources. This is the Enlightenment of the century.
+
+Sovereign individuals will be able to freely express themselves and transact with each other **without any kind of intermediary** exercising their unjustified power and oppressing them.
+
+This may be one of the most important revolutions humanity has ever faced. We need to be careful about how we go about it. Technology can be a double-edged sword. We need to explicitly **set out the values that Aragon stands for**, in order for our community to always **uphold and defend them**.
```
Comment (Smokyish) [inline docs/documentation/manifesto.md]: Bolding does not match the blog post

File: docs/documentation/manifesto.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+## We are committed to building organizational forms that defend self-sovereignty — where a user can always exercise choice, either by participating or exiting.
```
Comment (Smokyish) [inline docs/documentation/manifesto.md]: Should be bold

File: docs/documentation/manifesto.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+In today's world, our governing power over society's common goods is negligible.
+
+Even for exercising small changes over your environment (your city, state or corporation), you might have to go through cumbersome and opaque bureaucratic processes.
+
+This happens because the coordination costs were high in the old world. But now, thanks to the Internet and decentralized governance, we can, and must, engineer systems to empower the people's voice to their fullest potential.
+
+Also, if you dislike those broken governance processes, you **cannot easily exit from them**. Some organizations own your identity (governments), some others own your data (Facebook), and some others even own the path for you to exit (borders), restricting your choice of leaving.
```
Comment (Smokyish) [inline docs/documentation/manifesto.md]: Bolding doesn't match the blog post

File: docs/documentation/manifesto.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+
+**We must build organizational structures that allow users to exit with minimal friction** if they fundamentally disagree with the governance of those structures.
+
+## We are committed to creating collaboration mechanisms in which violence is not only disincentivized, but impossible.
```
Comment (Smokyish) [inline docs/documentation/manifesto.md]: Should be bold

File: docs/documentation/manifesto.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+
+Choices can only be made freely in the absence of coercion and extortion. **Systems should never use violence** as a means to incentivize or disincentivize human behavior.
+
+With cryptonetworks, we can create environments where others cannot even tell where individuals are located or what they look like. We must make violence in these cryptonetworks impossible by protecting users' privacy with cryptography.
+
+## We are committed to decentralizing power in order to dismantle unjustified power — which usually springs from centralization.
```
Comment (Smokyish) [inline docs/documentation/manifesto.md]: Should be bold

File: docs/documentation/manifesto.md [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+
+Power has a natural tendency to reinforce itself and become corrupt. If power becomes centralized, it doesn't have to answer to anyone but itself. **Decentralizing power is essential** to minimizing corruption over time.
+
+We must strive to create systems in which a large number of diverse stakeholders have a say, in order for common goods to be responsibly governed by their communities.
+
+## We are committed to the creation of long-term value versus short-term profit — which in turn, advances sustainability.
```
Comment (Smokyish) [inline docs/documentation/manifesto.md]: Should be bold

File: docs/documentation/manifesto.md [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+
+We want these values to last. **We are in this for the long run.** In order to make lasting impact and disrupt existing power structures, **we must create systems that provide sustainable economic value to their participants.**
+
+We must honor and encourage the communities that sustain Aragon itself, and we must do so by rewarding those who defend the values outlined in this manifesto.
+
+## We are committed to a world in which every person can participate in these new organizational structures.
```
Comment (Smokyish) [inline docs/documentation/manifesto.md]: Should be bold

File: docs/documentation/manifesto.md [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+
+We seek to use technology to lift people from oppression. To be successful, **we must keep our products open, understandable, and easy to use** for everyone.
+
+## [Aragon Foundation](https://aragon.org/foundation/) Pledge
+
+The [Aragon Foundation](https://aragon.org/foundation/) is a non-profit entity that safeguards the values of the project.
+
+**Specifically, the Aragon Foundation pledges to support the Aragon Manifesto by:**
+- Building free, libre, open source technology to allow the creation and management of decentralized organizations
```
Comment (Smokyish) [inline docs/documentation/manifesto.md]: Bullet list not working, needs an extra empty line

File: docs/documentation/manifesto.md [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+- Experimenting with new governance models that comply with the Manifesto's values
+- Creating a cryptonetwork for decentralized organizations to thrive (the [Aragon Network](https://aragon.org/network))
+- Building a community that defends the Manifesto's values
+- Using the token sale proceeds to fund teams that uphold and defend these values, in order for them to carry and accomplish all the aforementioned tasks
+- Working alongside other entities that pledge to support decentralized technologies and build the underlying infrastructure needed for decentralized organizations to flourish
+- Use all of its assets, including proceeds from the token sale, intellectual property, and brand to maintain, spread and defend the above values
+
+We consider this Manifesto **an alpha version**, and look forward to the [community's feedback](https://aragon.chat/) to evolve it over time to a definitive version.
```
Comment (Smokyish) [inline docs/documentation/manifesto.md]: Wording differs from the blog post and missing bolding

**Comment by john-light** [comment]
@Smokyish updated, thank you for the detailed review!

**Comment by Smokyish** [comment]
@john-light Added small formatting changes and the image to the page, good to go from my end.

**Pull request merged**